src/widgets/courses/ui/courses.module.scss

```diff
@@ -0,0 +1,42 @@
+.all-courses {
+  @include media-laptop {
+    padding: 40px;
+  }
+
+  @include media-tablet {
+    padding: 0;
+  }
+}
+
+.courses-content {
+  padding-top: 40px;
+
+  @include media-laptop {
+    flex-wrap: wrap;
+    gap: 24px;
+    width: 100%;
+    padding: 8px 6px;
+  }
+
+  @include media-tablet {
+    padding: 0 16px;
+  }
+}
```

Comment:
We need to use unified paddings for container and content in the application. It doesn't look good right now. 
![image](https://github.com/user-attachments/assets/7f8d7e72-cec4-41a0-b36b-af211db59f69)
![image](https://github.com/user-attachments/assets/66650b0d-e2b0-4a9b-8d48-b4fb4db6d4ff)
![image](https://github.com/user-attachments/assets/013fb49a-92ef-4ffb-95b4-88648990f1b3)

Reply:
Done